docs/CustomCommands.md

```diff
@@ -96,3 +98,71 @@ it('execute external library in a sync way', () => {
 ```
 
 Note that the result of your custom command will be the result of the promise you return. Also there is no support for synchronous commands in standalone mode therefore you always have to handle asynchronous commands using promises.
+
+### Overwriting native commands
+
+You can also overwrite native commands with `overwriteCommand`. It is highly unrecommended to do that because it may lead to unpredictable behavior of framework! Overall approach is similar to `addCommand`, the only difference is that first argument in command function is original function, please see some examples below.
```

Comment:
```suggestion
You can also overwrite native commands with `overwriteCommand`. It is not recommended to do this because it may lead to unpredictable behavior of the framework! The overall approach is similar to `addCommand`, the only difference is that the first argument in the command function is the original function that you are about to overwrite. Please see some examples below:
```